src/org/newdawn/slick/openal/SoundStore.java

```diff
@@ -52,7 +52,6 @@
  * @author Kevin Glass
  * @author Rockstar setVolume cleanup 
  */
-@SuppressWarnings({"rawtypes", "unchecked", "unused"})

```

Comment:
Don't edit the files I'm overriding from third-party libraries -- the more changes, the harder it is to diff against the originals.